code/game/objects/items/ego_weapons/teth.dm

```diff
@@ -126,7 +126,7 @@
 /obj/item/ego_weapon/regret/melee_attack_chain(mob/user, atom/target, params)
 	..()
 	user.changeNext_move(CLICK_CD_MELEE * 2) // Really Slow. This is the slowest teth we have, +0.4 to Eyes 1.6
```

Comment:
While you are at it, this probably shouldn't be there?

Reply:
This was added in-between the attack speed refactor. Might want to convert it to that system